src/coreclr/ToolBox/superpmi/superpmi-shared/lightweightmap.h

```diff
@@ -319,7 +319,7 @@ class LightWeightMap : public LightWeightMapBuffer
 
         // If we have RTTI, we can make this assert report the correct type. No RTTI, though, when
         // built with .NET Core, especially when built against the PAL.
-        AssertCodeMsg((ptr - bytes) == size, EXCEPTIONCODE_LWM, "%s - Ended with unexpected sizes %p != %x",
+        AssertCodeMsg((unsigned int)(ptr - bytes) == size, EXCEPTIONCODE_LWM, "%s - Ended with unexpected sizes %p != %x",
```

Comment:
I would rewrite it as follows to avoid casting
```suggestion
        AssertCodeMsg(ptr == (bytes + size), EXCEPTIONCODE_LWM, "%s - Ended with unexpected sizes %p != %x",
```